mypy/stubtest.py

```diff
@@ -1679,16 +1680,22 @@ def get_importable_stdlib_modules() -> set[str]:
         all_stdlib_modules = sys.stdlib_module_names
     else:
         all_stdlib_modules = set(sys.builtin_module_names)
```

Comment:
@hauntsaninja we start off with the `sys.builtin_module_names` modules being in the set

Reply:
Oh oops, I missed that. Thanks!

Reply:
No worries — thanks for the review! :D